File: learn/drill/programming_in_th/0004/0004.c
#include <stdio.h>

int main(void)
{
    char s[10001];
    int cases[] = { 0, 0 };

    scanf("%s", s);

    for (int i = 0; s[i] != '\0'; ++i)
    {
        if (s[i] <= 90)
        {
            cases[0] = 1;
        }
        else
        {
            cases[1] = 1;
        }
    }

    if (cases[0] && cases[1])
    {
        printf("Mix\n");
    }
    else if (cases[0])
    {
        printf("All Capital Letter\n");
    }
    else
    {
        printf("All Small Letter\n");
    }

    return 0;
}

File: learn/drill/programming_in_th/0004/0004.py
s = input()
res = [0, 0]

for c in s:
    if c.isupper():
        res[0] = 1
    else:
        res[1] = 1

if res[0] and res[1]:
    print("Mix")
elif res[0]:
    print("All Capital Letter")
else:
    print("All Small Letter")